src/components/SingleDatePicker.jsx

```diff
@@ -248,7 +248,7 @@ class SingleDatePicker extends React.PureComponent {
 
   onFocusOut(e) {
     const { onFocusChange } = this.props;
-    if (this.dayPickerContainer.contains(e.relatedTarget || e.target)) return;
+    if (this.dayPickerContainer.contains(e.relatedTarget || e.target) || this.dayPickerContainer.contains(e.target)) return;
```

Comment:
Can we add a test and/or story to cover this?

It seems like if `e.relatedTarget` is falsy, this line will cause `contains` to be invoked twice with the same `e.target`. Can that be avoided?